spec/namespaces/snapshot.yaml

```diff
@@ -335,30 +335,51 @@ components:
               rename_pattern:
                 type: string
+                description: The pattern to apply to the restored data streams and indices. Data streams and indices matching the rename pattern will be renamed according to the `rename_replacement` setting.  The rename pattern is applied as defined by the regular expression that supports referencing the original text.   The request fails if two or more data streams or indices are renamed into the same name. If you rename a restored data stream, its backing indices are also renamed. For example, if you rename the logs data stream to `recovered-logs`, the backing index `.ds-logs-1` is renamed to `.ds-recovered-logs-1`.  If you rename a restored stream, ensure an index template matches the new stream name. If there are no matching index template names, the stream cannot roll over and new backing indices are not created.
               rename_replacement:
                 type: string
+                description: The rename replacement string.
+              rename_alias_pattern:
+                x-version-added: '2.18'
+                type: string
+                description: The pattern to apply to the restored aliases. Aliases matching the rename pattern will be renamed according to the `rename_alias_replacement` setting. The rename pattern is applied as defined by the regular expression that supports referencing the original text.  If two or more aliases are renamed into the same name, these aliases will be merged into one.
+              rename_alias_replacement:
+                x-version-added: '2.18'
+                type: string
+                description: The rename replacement string for aliases.
+              source_remote_store_repository:
+                x-version-added: '2.10'
+                type: string
+                description: The name of the remote store repository of the source index being restored. If not provided, the Snapshot Restore API will use the repository that was registered when the snapshot was created.
+              storage_type:
+                x-version-added: '2.7'
+                type: string
+                description: '`local` indicates that all snapshot metadata and index data will be downloaded to local storage.  `remote_snapshot` indicates that snapshot metadata will be downloaded to the cluster, but the remote repository will remain the authoritative store of the index data. Data will be downloaded and cached as necessary to service queries. At least one node in the cluster must be configured with the search role in order to restore a snapshot using the type `remote_snapshot`.  Defaults to `local`.'
```

Comment:
```suggestion
                description: |-
                     Storage type.
                     A value of `local` indicates that all snapshot metadata and index data will be downloaded to local storage.
                     A value of `remote_snapshot` indicates that snapshot metadata will be downloaded to the cluster, but the remote repository will remain the authoritative store of the index data.
                     Data will be downloaded and cached as necessary to service queries. At least one node in the cluster must be configured with the search role in order to restore a snapshot using the type `remote_snapshot`.
                     Defaults to `local`.
```

Reply:
I added line breaks to match the [source documentation](https://github.com/opensearch-project/documentation-website/blob/main/_api-reference/snapshots/restore-snapshot.md).